Update octoprint.markdown (#22925)
home-assistant/home-assistant.github.io: source/_integrations/octoprint.markdown

{% endconfiguration_basic %}

### API Key

> For the integration to work, please check that the plugin Discovery is enabled. note: make sure to enable pop-ups in the settings -> printer notifications menu.

Reviewer: ```suggestion
For the integration to work, please check that in Octoprint, the plugin Discovery is enabled and in the settings -> printer notifications menu pop-ups are enabled.
```

### API Key
For the integration to work, please check that the plugin Discovery is enabled.

> The Octoprint integration will attempt to register itself via the [application keys plugin](https://docs.octoprint.org/en/master/bundledplugins/appkeys.html). After submitting the configuration UI in Home Assistant, open the Octoprint UI and click allow on the prompt. note: make sure to enable pop-ups in the settings -> printer notifications menu.

Reviewer: I think this makes more sense as part of the first sentence (which would incorporate it more into the existing text, instead of putting an in-line note at the end).

Author: agree. thi sis my first time using github. so bear with me. i am still learning how to do this.
